src/lib/Libifl/pbs_loadconf.c

```diff
@@ -919,6 +924,22 @@ __pbs_loadconf(int reload)
 			goto err;
 		}
 	}
+	if (pbs_conf.cp_path == NULL) {
+
+#ifdef WIN32
+		if ((pbs_conf.cp_path =
+			malloc(6 * sizeof(char))) != NULL) {
```

Comment:
Some editors get wonky when braces don't match due to ifdefs.  I suggest you move the malloc() on top of the ifdef and malloc 8 bytes.  You can do your error checking then.  The only thing in the ifdef will be the sprintf() for each platform.

Reply:
Suggested changes have been committed and pushed.

Reply:
how about using `strdup()` instead of `malloc()+sprintf()` like below:
```c
if (pbs_conf.cp_path == NULL) {
#ifndef WIN32
   char *cmd = "/bin/cp";
#else
   char *cmd = "xcopy";
#endif
   pbs_conf.cp_path = strdup(cmd);
   if (pbs_conf.cp_path == NULL)
       goto err;
}

Reply:
It doesn't make much difference, but `strdup()` is a bit easier to read. Checking #ifdef WIN32 is more consistent than checking #ifndef. 

`	if (pbs_conf.cp_path == NULL) {
#ifdef WIN32
		char *cmd = 'xcopy';
#else
		char *cmd = "/bin/cp";
#endif
		pbs_conf.cp_path = strdup(cmd);
		if (pbs_conf.cp_path == NULL) {
			goto err;
		}
	}`